src/Phinx/Migration/Manager.php

```diff
@@ -158,7 +160,9 @@ public function printStatus($environment, $format = null)
                 // this means we have no up migrations, so we write all the missing versions already so they show up
                 // before any possible down migration
                 foreach ($missingVersions as $missingVersionCreationTime => $missingVersion) {
-                    $this->printMissingVersion($missingVersion, $maxNameLength);
+                    if ($format !== 'json') {
+                        $this->printMissingVersion($missingVersion, $maxNameLength);
+                    }
```

Comment:
Given how much this code is repeated, can we add the check for the format in the function itself?

Reply:
Yeah, these if statements were a clumsy way to go about it. I've implemented a vastly simpler solution.